### ISSUE TITLE
(Question) Change to new Google UI template?

### ISSUE THREAD

**Issue by MrCoolAndroid** [yuliskov/SmartTube#2654]
Note: I wasn't able to give a feedback with the QR in the app because the QR code was way on the border and I could not scan it so I thought that I could give the feedback here, please delete this issue if needed!

Hey! A couple of weeks ago I saw that Google might be releasing a new UI template (assembling more to the newest Android versions and Google TV) and I was thinking, if this template changes will you update the UI according to it? (Not a request or anything, just a question)

**Comment by arunkd13** [comment]
What UI templates are your referring to? Do you have any links to the Google website which gives more details?

**Comment by MrCoolAndroid** [comment]
@arunkd13 Sure! Here's an [article on the Android Developer Blog](https://android-developers.googleblog.com/2023/05/building-pixel-perfect-living-room-experiences-compose-for-tv.html?m=1), I actually didn't know it was called "Compose for TV", I should have specified

**Comment by arunkd13** [comment]
Great, thanks for the pointer. I cannot answer for the project owners, but looks like it would be the right direction for a UI refresh.

**Comment by MrCoolAndroid** [comment]
Yeah, when I saw about Compose for TV I thought this app could use an UI refresh with this new template
